### ISSUE TITLE
Landing page

### ISSUE THREAD

**Issue by LERoise** [Itera/mad-learning-web#10]
### Features
* Use Event List component to list out coming events

**Comment by anhtin** [comment]
Assuming that the backend handles the connection with the database, this issue – along with #2, #3, and #4 – is partly blocked by #16. Should we perhaps discuss and agree on an inferface between the frontend and the backend? It would allow us to temporarily work with mock data on the frontend until the backend is up and running.

**Comment by LERoise** [comment]
WIP wireframes, you can ignore the search field now for the MVP since its out of scope:
![image](https://user-images.githubusercontent.com/3605498/93870303-51daa400-fccd-11ea-99e4-d851fd1f0e1b.png)
